### PR TITLE
new readme

### PR DESCRIPTION
**Pull request by SophieDeBenedetto**
need some :eyes: 

closes issue #1 
@dfenjves @ipc103 @aviflombaum @deniznida

### CODE REVIEW

**Comment by AnnJohn** [comment]
@dfenjves Do you want to review this one? I'll assign and set a deadline.

**Comment by dfenjves** [comment]
Sure

**Comment by AnnJohn** [comment]
@dfenjves Thanks! Deadline- 9/10/15 if you can't get to it today.

File: README.md [hunk 1]
```diff
@@ -11,8 +5,82 @@
...
+
+```bash
+which sqlite3
+```
+
+if you get back
+
+`/usr/bin/sqlite3`
+
+Then you have a working version of sqlite3 already installed on your system. Thanks Apple! Skip ahead to
```
Comment (dfenjves) [inline README.md]: Maybe "Skip ahead to the next lesson" or "mark this lesson as complete and move on"?

**Comment by dfenjves** [comment]
Looks great. Apart from the little "skip ahead note" this is ready for merge.

**Pull request merged**